enhancements/installer/single-node-installation-bootstrap-in-place.md

```diff
@@ -0,0 +1,482 @@
+---
+title: single-node-deployment-with-bootstrap-in-place
+authors:
+  - "@eranco"
+  - "@mrunalp"
+  - "@dhellmann"
+  - "@romfreiman"
+  - "@tsorya"
+reviewers:
+  - TBD
+  - "@markmc"
+  - "@deads2k"
+  - "@wking"
+  - "@eparis"
+  - "@hexfusion"
+approvers:
+  - TBD
+creation-date: 2020-12-13
+last-updated: 2020-12-13
+status: implementable
+see-also:
+  - https://github.com/openshift/enhancements/pull/560
+  - https://github.com/openshift/enhancements/pull/302
+---
+
+# Single Node deployment with bootstrap-in-place
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+As we add support for new features such as [single-node production deployment](https://github.com/openshift/enhancements/pull/560/files),
+we need a way to install such clusters without an extra node dependency for bootstrap.
+
+This enhancement describes the flow for installing Single Node OpenShift using a liveCD that performs the bootstrap logic and reboots to become the single node.
+
+## Motivation
+
+Currently, all OpenShift installations use an auxiliary bootstrap node.
+The bootstrap node creates a temporary control plane that is required for launching the actual cluster.
+
+Single Node OpenShift installations will often be performed in environments where there are no extra nodes, so it is highly desirable to remove the need for a separate bootstrap machine to reduce the resources required to install the cluster.
+
+The downsides of requiring a bootstrap node for Single Node OpenShift are:
+
+1. The obvious additional node.
+2. Requires external dependencies:
+   1. Load balancer (only for bootstrap phase)
+   2. Preconfigured DNS (for each deployment)
+3. Cannot use Bare Metal IPI:
```

Comment:
You are right,  it's confusing.
Bare Metal IPI is an option that can mitigate the external dependencies described in 2.
When using BM platform you won't need external LB and DNS since the platform takes care of it for you.
The point I was trying to make is that it's not a good option since it comes with a price of more dependencies and components we don't need to run post installation (VIPs, keepalived, mdns).